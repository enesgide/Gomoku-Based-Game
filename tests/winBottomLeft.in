place A19
place A18
place B18
place B17
place C17
place C16
place D16
place D15
place E15