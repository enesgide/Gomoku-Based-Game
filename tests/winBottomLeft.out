Black wins!
A19A18B18B17C17C16D16D15E15
Thank you for playing!
